components/brave_wallet/browser/brave_wallet_provider_impl.cc

```diff
@@ -7,15 +7,50 @@
 
 #include <utility>
 
+#include "base/json/json_reader.h"
+#include "base/json/json_writer.h"
+#include "base/strings/stringprintf.h"
 #include "brave/components/brave_wallet/browser/brave_wallet_provider_delegate.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_utils.h"
 #include "brave/components/brave_wallet/browser/eth_json_rpc_controller.h"
+#include "brave/components/brave_wallet/browser/eth_response_parser.h"
+#include "brave/components/brave_wallet/common/value_conversion_utils.h"
+#include "brave/components/brave_wallet/common/web3_provider_constants.h"
+#include "brave/components/brave_wallet/renderer/brave_wallet_response_helpers.h"
```

Comment:
I don't think you should be including something from the renderer here in the browser layer.
Either we should move it to common or you could do this on the renderer side. It'd be ok not to localize those strings I think if you decide to put it on the renderer side.

Reply:
fixed already, I moved it to renderer in last commit